contracts/staking/StakingManager.sol

```diff
@@ -144,12 +139,7 @@ abstract contract StakingManager is
         // Ensure the nodeID is not the zero address, and is not already an active validator.
         require(nodeID != bytes32(0), "StakingManager: Invalid node ID");
         require($._activeValidators[nodeID] == bytes32(0), "StakingManager: Node ID already active");
-
-        // Ensure the signature is the proper length. The EVM does not provide an Ed25519 precompile to
-        // validate the signature, but the P-Chain will validate the signature. If the signature is invalid,
-        // the P-Chain will reject the registration, and the stake can be returned to the staker after the registration
-        // expiry has passed.
-        require(signature.length == 64, "StakingManager: Invalid signature length");
+        require(blsPublicKey.length == 48, "StakingManager: Invalid blsPublicKey length");
```

Comment:
can have a constant for the length

Reply:
Since these packing/unpacking methods are likely to be refactored/optimized significantly in the future, I'm going to hold off on making further changes for now. Also pointing out that there were no length constants in the previous implementation.